module4/projects/ApiCheckpoint.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: page
+title: API Checkpoint
+---
+
+# Take Home Challenge
+
+A **Take-Home Challenge** is a common part of the interview process for applicants, especially beginner developers, applying for their first job in programming or software development. It is a practical task or project that will help the employer assess a candidate's technical skills, problem-solving abilities, and coding proficiency.
+
+Typically, the take-home challenge involves a specific assignment related to the position or the company's domain. The candidate is given a set of instructions, requirements, and sometimes sample data to work with. They are expected to complete the task within a specified time frame, which could range from a few days to a week, depending on the complexity of the challenge.
+
+Once the employer reviews the submission, they typically schedule a follow-up interview to discuss the candidate's solution, reasoning, and design choices.
+
+## The Challenge
+
+### Objective:
+The purpose of this take-home challenge is to assess your ability to design, implement, and test a RESTful API in C#/.NET that manages a collection of products associated with a "Merchant" resource.
+
+### Task Description:
+You are provided with an existing API for a **Merchant** resource, and your task is to build a C#/.NET API to manage a **Products** resource, where each product belongs to a specific merchant. The API should support basic CRUD operations (Create, Read, Update, Delete) for Products. Additionally, you need to implement XXXX statistics endpoints that provide valuable insights into the product collection as a whole, and for each merchant.
+
+### Setup
+
+* Fork and clone this [starter repo](https://github.com/turingschool-examples/Launch_Mod4TakeHomeChallenge)
```

Comment:
Left comments in slack, but I'll add them here too:
* When I start up the app it takes me to https://localhost:7136/weatherforecast in the browser, not sure where this is configured
* I was able to POST and create a merchant and GET and get all merchants without changing the code :tada:
* I did have to click disable SSL Verification in Postman though